geomesa-convert/geomesa-convert-all/src/test/scala/org/locationtech/geomesa/convert/testing/CompositeTextConverterTest.scala

```diff
@@ -40,7 +40,7 @@ class CompositeTextConverterTest extends Specification with LazyLogging {
       |   type         = "composite-converter",
       |   converters = [
       |     { converter = "first",   predicate = "strEq('1', trim(substr($0, 0, 2)))" },
-      |     { converter = "second",  predicate = "strEq('2', trim(substr($0, 0, 2)))" }
+      |     { converter = "second",  predicate = "boolEq(true, contains($0, 'world'))" }
```

Comment:
Does this need to be strContains to distinguish between spatial contains?

Reply:
these are transformer functions, we don't currently have any spatial functions exposed.